[2024-06-07T09:14:14.486+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-07T09:14:14.500+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: f1_data_pipeline_taskflow_with_kafka.consume_and_load_to_postgres manual__2024-06-07T09:14:12.217986+00:00 [queued]>
[2024-06-07T09:14:14.504+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: f1_data_pipeline_taskflow_with_kafka.consume_and_load_to_postgres manual__2024-06-07T09:14:12.217986+00:00 [queued]>
[2024-06-07T09:14:14.504+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-07T09:14:14.511+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): consume_and_load_to_postgres> on 2024-06-07 09:14:12.217986+00:00
[2024-06-07T09:14:14.516+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=71) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-07T09:14:14.517+0000] {standard_task_runner.py:63} INFO - Started process 73 to run task
[2024-06-07T09:14:14.517+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'f1_data_pipeline_taskflow_with_kafka', 'consume_and_load_to_postgres', 'manual__2024-06-07T09:14:12.217986+00:00', '--job-id', '85', '--raw', '--subdir', 'DAGS_FOLDER/kafkaProduce.py', '--cfg-path', '/tmp/tmp7tfq9dwn']
[2024-06-07T09:14:14.519+0000] {standard_task_runner.py:91} INFO - Job 85: Subtask consume_and_load_to_postgres
[2024-06-07T09:14:14.549+0000] {task_command.py:426} INFO - Running <TaskInstance: f1_data_pipeline_taskflow_with_kafka.consume_and_load_to_postgres manual__2024-06-07T09:14:12.217986+00:00 [running]> on host 9459748fcbed
[2024-06-07T09:14:14.601+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='f1_data_pipeline_taskflow_with_kafka' AIRFLOW_CTX_TASK_ID='consume_and_load_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2024-06-07T09:14:12.217986+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-07T09:14:12.217986+00:00'
[2024-06-07T09:14:14.602+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-07T09:14:14.603+0000] {conn.py:362} INFO - <BrokerConnection node_id=bootstrap-0 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]>: connecting to kafka:9092 [('172.18.0.6', 9092) IPv4]
[2024-06-07T09:14:14.604+0000] {conn.py:957} INFO - Probing node bootstrap-0 broker version
[2024-06-07T09:14:14.604+0000] {conn.py:393} INFO - <BrokerConnection node_id=bootstrap-0 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]>: Connection complete.
[2024-06-07T09:14:14.707+0000] {conn.py:1019} INFO - Broker version identified as 2.6.0
[2024-06-07T09:14:14.708+0000] {conn.py:1020} INFO - Set configuration api_version=(2, 6, 0) to skip auto check_version requests on startup
[2024-06-07T09:14:14.709+0000] {consumer.py:122} WARNING - group_id is None: disabling auto-commit.
[2024-06-07T09:14:14.709+0000] {subscription_state.py:167} INFO - Updating subscribed topics to: ('f1_data_topic',)
[2024-06-07T09:14:14.710+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2024-06-07T09:14:14.711+0000] {base.py:84} INFO - Using connection ID 'sourcedb_connection' for task execution.
[2024-06-07T09:14:14.715+0000] {subscription_state.py:253} INFO - Updated partition assignment: [('f1_data_topic', 0)]
[2024-06-07T09:14:14.717+0000] {conn.py:362} INFO - <BrokerConnection node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]>: connecting to kafka:9092 [('172.18.0.6', 9092) IPv4]
[2024-06-07T09:14:14.717+0000] {conn.py:393} INFO - <BrokerConnection node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]>: Connection complete.
[2024-06-07T09:14:14.717+0000] {conn.py:673} INFO - <BrokerConnection node_id=bootstrap-0 host=kafka:9092 <connected> [IPv4 ('172.18.0.6', 9092)]>: Closing connection. 
[2024-06-07T09:21:55.116+0000] {conn.py:850} ERROR - <BrokerConnection node_id=1 host=kafka:9092 <connected> [IPv4 ('172.18.0.6', 9092)]>: socket disconnected
[2024-06-07T09:21:55.117+0000] {conn.py:673} ERROR - <BrokerConnection node_id=1 host=kafka:9092 <connected> [IPv4 ('172.18.0.6', 9092)]>: Closing connection. KafkaConnectionError: socket disconnected
[2024-06-07T09:21:55.118+0000] {client_async.py:330} WARNING - Node 1 connection failed -- refreshing metadata
[2024-06-07T09:21:55.119+0000] {future.py:77} ERROR - Fetch to node 1 failed: KafkaConnectionError: socket disconnected
[2024-06-07T09:21:55.120+0000] {conn.py:362} INFO - <BrokerConnection node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]>: connecting to kafka:9092 [('172.18.0.6', 9092) IPv4]
[2024-06-07T09:21:55.121+0000] {conn.py:402} ERROR - Connect attempt to <BrokerConnection node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]> returned error 111. Disconnecting.
[2024-06-07T09:21:55.122+0000] {conn.py:673} ERROR - <BrokerConnection node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2024-06-07T09:21:55.122+0000] {client_async.py:330} WARNING - Node 1 connection failed -- refreshing metadata
[2024-06-07T09:21:55.224+0000] {conn.py:362} INFO - <BrokerConnection node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]>: connecting to kafka:9092 [('172.18.0.6', 9092) IPv4]
[2024-06-07T09:21:55.225+0000] {conn.py:402} ERROR - Connect attempt to <BrokerConnection node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]> returned error 111. Disconnecting.
[2024-06-07T09:21:55.225+0000] {conn.py:673} ERROR - <BrokerConnection node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2024-06-07T09:21:55.226+0000] {client_async.py:330} WARNING - Node 1 connection failed -- refreshing metadata
[2024-06-07T09:21:55.428+0000] {conn.py:362} INFO - <BrokerConnection node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]>: connecting to kafka:9092 [('172.18.0.6', 9092) IPv4]
[2024-06-07T09:21:55.428+0000] {conn.py:402} ERROR - Connect attempt to <BrokerConnection node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]> returned error 111. Disconnecting.
[2024-06-07T09:21:55.428+0000] {conn.py:673} ERROR - <BrokerConnection node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2024-06-07T09:21:55.429+0000] {client_async.py:330} WARNING - Node 1 connection failed -- refreshing metadata
[2024-06-07T09:21:55.882+0000] {conn.py:362} INFO - <BrokerConnection node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]>: connecting to kafka:9092 [('172.18.0.6', 9092) IPv4]
[2024-06-07T09:21:55.883+0000] {conn.py:402} ERROR - Connect attempt to <BrokerConnection node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]> returned error 111. Disconnecting.
[2024-06-07T09:21:55.883+0000] {conn.py:673} ERROR - <BrokerConnection node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2024-06-07T09:21:55.883+0000] {client_async.py:330} WARNING - Node 1 connection failed -- refreshing metadata
[2024-06-07T09:21:56.588+0000] {conn.py:362} INFO - <BrokerConnection node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]>: connecting to kafka:9092 [('172.18.0.6', 9092) IPv4]
[2024-06-07T09:21:56.589+0000] {conn.py:402} ERROR - Connect attempt to <BrokerConnection node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]> returned error 111. Disconnecting.
[2024-06-07T09:21:56.590+0000] {conn.py:673} ERROR - <BrokerConnection node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2024-06-07T09:21:56.590+0000] {client_async.py:330} WARNING - Node 1 connection failed -- refreshing metadata
[2024-06-07T09:21:57.546+0000] {conn.py:362} INFO - <BrokerConnection node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]>: connecting to kafka:9092 [('172.18.0.6', 9092) IPv4]
[2024-06-07T09:21:57.547+0000] {conn.py:402} ERROR - Connect attempt to <BrokerConnection node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]> returned error 111. Disconnecting.
[2024-06-07T09:21:57.547+0000] {conn.py:673} ERROR - <BrokerConnection node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2024-06-07T09:21:57.547+0000] {client_async.py:330} WARNING - Node 1 connection failed -- refreshing metadata
[2024-06-07T09:21:58.554+0000] {conn.py:362} INFO - <BrokerConnection node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]>: connecting to kafka:9092 [('172.18.0.6', 9092) IPv4]
[2024-06-07T09:21:58.554+0000] {conn.py:402} ERROR - Connect attempt to <BrokerConnection node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]> returned error 111. Disconnecting.
[2024-06-07T09:21:58.555+0000] {conn.py:673} ERROR - <BrokerConnection node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2024-06-07T09:21:58.555+0000] {client_async.py:330} WARNING - Node 1 connection failed -- refreshing metadata
[2024-06-07T09:22:03.613+0000] {conn.py:362} INFO - <BrokerConnection node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]>: connecting to kafka:9092 [('172.18.0.6', 9092) IPv4]
[2024-06-07T09:22:03.613+0000] {conn.py:393} INFO - <BrokerConnection node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]>: Connection complete.
